lib/pure/random.nim

```diff
@@ -635,20 +653,60 @@ when not defined(nimscript) and not defined(standalone):
       let time = int64(times.epochTime() * 1000) and 0x7fff_ffff
       result = initRand(time)
     else:
-      let now = times.getTime()
-      result = initRand(convert(Seconds, Nanoseconds, now.toUnix) + now.nanosecond)
+      proc getRandomState(): Rand =
+        when defined(nimscript):
```

Comment:
how about adding a vmops for this, that way you don't need to special case nimscript with a suboptimal seeding, and it then also works for VM instead of having to document that it doesn't work